src/systems/particle_emitter/ParticleEmitter.hh

```diff
@@ -96,7 +96,10 @@ namespace systems
   ///                        therefore only the horizontal dimension of the
   ///                        image is used.
   ///                        Note that this function overrides the particle
-  /// colors set with <color_start> and <color_end>.
+  ///                        colors set with <color_start> and <color_end>.
+  /// `<topic>`: Topic used to update particle emitter properties at runtime.
+  ///            The default topic is
+  ///            /model/<model_name>/particle_emitter/<emitter_name>
```

Comment:
Should we add some documentation about the `particleEmitterCmd` option for updating the particle emitter at runtime? In particular, I think we should document at least the following:
1. How to assign a `particleEmitterCmd` to a particular particle emitter (I'm assuming the `name` field of the command should match the `name` field used when creating the original emitter)
2. Which properties of the emitter can be changed at runtime (in other words, which fields can be set in `particleEmitterCmd` that would modify the emitter associated with the command)

Reply:
added some brief doc and referred the user to look at the example world file. f612ec8